components/zenFunctions/server/fn_zen_doCreateJammer.sqf

```diff
@@ -0,0 +1,37 @@
+#include "macros.hpp"
+/*
```

Comment:
BTW I'd also like if you could put these comments above all code and throw an empty line between them and any code tyvm